igcollect/linux_disklatency.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+"""igcollect - Linux Disk I/O latency
+
+Copyright (c) 2019 InnoGames GmbH
+"""
+
+from argparse import ArgumentParser
+from subprocess import check_output
+from time import time
+import sys
+from libvirt import (
+    open,
```

Comment:
Libvirts choice of name here is unfortunate. Every halfway sane python programmer will assume a call to open() in python code to reference pythons buildin function to open files. This both makes this buildin inaccessible and confusing.  Change this to something like `from libvirt import open as libvirt_open, libvirtError`

Reply:
true, updated. thanks